README.md

```diff
@@ -12,7 +12,8 @@ a corresponding JavaScript API:
 * `Sec-CH-UA-Platform`
 * `Sec-CH-UA-Platform-Version`
 * `Sec-CH-UA`
-* `Sec-CH-UA-Full-Version`
+* `Sec-CH-UA-Full-Version` (deprecated in favor of `Sec-CH-UA-Version-List`)
```

Comment:
Do we have use counters for `Full-Version`? Could we just change its semantics in place (instead of rename)?

Reply:
Let me check on that. It would be nicer if we could...

Reply:
OK, we might have missed the boat here with 1.6% of page loads, sadly 😿:

https://chromestatus.com/metrics/feature/timeline/popularity/3189

<img width="730" alt="Screen Shot 2021-08-18 at 1 46 53 PM" src="https://user-images.githubusercontent.com/67283/129946967-c89c0138-5df9-4187-8a58-f205cdc334f6.png">

Reply:
@yoavweiss looks like it's too high to change here. Shall we continue review assuming a new hint?

Reply:
1.6% is an astoundingly high number. Do you have any insight into who's using this mechanism? Is there some large embedded resource that's causing this to show up on lots of sites, or are there large top-level sites opting in?

Reply:
We know that YouTube has been experimenting with UA-CH (and sadly [we caused a crash there, oops](https://bugs.chromium.org/p/chromium/issues/detail?id=1217589#c17)), and we have some other partners who have committed to doing some testing.